distribution/lib/Standard/Base/0.0.0-dev/src/Polyglot/Proxy_Polyglot_Array.enso

```diff
@@ -0,0 +1,23 @@
+from Standard.Base import Polyglot
+
+## Advanced
+
+    Wrapper for Polyglot Arrays
+type Proxy_Polyglot_Array
```

Comment:
we should also move the `to_array` into this and change the field to be called `storage`.

This is to enable using a slice or a Table as the backing storage for the Vector.